encoder/vaapiencoder_vp8.cpp

```diff
@@ -75,6 +75,10 @@ void VaapiEncoderVP8::resetParams()
 {
     //5 times compress ratio
     m_maxCodedbufSize = width() * height() * 3 / 2;
+    //when the resolution of video is very small such as 96x96 and buffer plus the header of vp8,
+    //the size of actual output buffer maybe be bigger the m_maxCodedbufSize.
+    //So plusing 4096 avoids the situation.
+    m_maxCodedbufSize += 0x1000;
```

Comment:
"Yes, But it is only an experience value for him, 48x188 should have same problem, and it may related to the content. If the content is complex, it should have problem for high resolution."

Can @clearlylin experiment with other values or try to find the root cause of the problem?  Is this a driver requirement?  If so, it would be good to fix it at the driver level instead.